packages/apps/docs/src/components/MostPopular/MostPopular.tsx

```diff
@@ -0,0 +1,26 @@
+import { BrowseSection } from '../BrowseSection';
+
+import { useFetchMostPopularPages } from './useFetchMostPopularPages';
+
+import Link from 'next/link';
+import React from 'react';
+
+const MostPopular = (): JSX.Element => {
```

Comment:
can you use FC (more inline with the other components (also with the react-ui lib components))